chat-plugins/hangman.js

```diff
@@ -225,7 +225,7 @@ exports.commands = {
 
 		guess: function (target, room, user) {
 			if (!target) return this.parse('/help guess');
-			if (!room.game || room.game.gameType !== 'hangman') return this.errorReply("There is no game of hangman running in this room.");
+			if (!room.game || room.game.title !== 'Hangman') return this.errorReply("There is no game of hangman running in this room.");
```

Comment:
Also, you might want to ducktype this. i.e. check for `room.game.guess` and then put most of the logic inside `room.game.guess()`.

Really, though.